retrieve/rep/event.go

```diff
@@ -0,0 +1,193 @@
+package rep
```

Comment:
I don't think we're actually subscribing to these events anywhere so we can probably just delete this entire rep package

Reply:
I think we do for the query ask. I tried to remove it once and I failed, I could give it another shot.

Reply:
Removed rep.